src/main/java/com/nexabra/socialplugin/commands/Friends.java

```diff
@@ -0,0 +1,252 @@
+package com.nexabra.socialplugin.commands;
+
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.ProxyServer;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.ComponentBuilder;
+import net.md_5.bungee.api.chat.TextComponent;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+import net.md_5.bungee.api.event.TabCompleteEvent;
+import net.md_5.bungee.api.plugin.Command;
+import com.nexabra.socialplugin.FriendsManager;
+import net.md_5.bungee.api.plugin.Plugin;
+import net.md_5.bungee.api.plugin.TabExecutor;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Objects;
+
+
+public class Friends extends Command {
+
+    public Friends() {
+        super("Friend", "", "f");
+    }
+
+    @Override
+    public void execute(CommandSender commandSender, String[] args) {
+
+        if (args.length < 1) {
+            sendHelpMenu(commandSender);
+            return;
+        }
+        if ((commandSender instanceof ProxiedPlayer)) {
+            ProxiedPlayer player = (ProxiedPlayer)commandSender;
```

Comment:
```suggestion
        if (commandSender instanceof final @NotNull ProxiedPlayer player) {
```
i saw ur using java 14 so actually idk if this is supported in that version